app/src/main/resources/application.properties

```diff
@@ -12,4 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-quarkus.log.level=${KAFKAPROXY_LOG_LEVEL:INFO}
\ No newline at end of file
+quarkus.log.level=${KAFKAPROXY_LOG_LEVEL:INFO}
+%dev.quarkus.log.console.json=false
```

Comment:
How about this?
`quarkus.log.console.json=${KAFKAPROXY_ENABLE_JSON_LOGGING:false}`

Reply:
Fine, let's do it so.

Reply:
Nice :-) Can you adjust the PR accordingly?

Reply:
I'll do and also adjust the Readme with documentation for the new Env var.